server/server.js

```diff
@@ -75,6 +75,19 @@ const appReady = app.prepare().then(() => {
     next()
   })
 
+  // robots.txt
+  if (process.env.NODE_ENV === 'production') {
```

Comment:
This is a good option. However, the test should be for process.env.APP_URL includes ('live')

Reply:
Really? What if someone else deploys the platform to not live?

Reply:
if not live - then don't allow anything to be crawled. 'Disallow: /'